site/app/templates/polls/NewPollPage.twig

```diff
@@ -1,451 +0,0 @@
-                            $("#response_" + i + "_wrapper").attr("id", "response_" + (i - 1) + "_wrapper");
-                        }
-                        my_this.parent().remove();
-                        count -= 1;
-                        $("#response-count").val("" + count);
-                        let temp = $("#removed-responses").val();
-                        if (temp !== "") {
-                            temp += ",";
-                        }
-                        temp += $(my_this.siblings(".option_id")[0]).attr("value");
-                        $("#removed-responses").val(temp);
-                    }
-                },
-                error: function(e) {
-                    alert("Error occurred when requesting via ajax. Please refresh the page and try again.");
-                }
-            });
-        });
-    }
-
-    $(document).ready(function() {
-        {% if poll is not defined %}
-            const today = new Date();
-            const year = today.getFullYear().toString().padStart(4, '0');
-            const month = (today.getMonth() + 1).toString().padStart(2, '0');
-            const day = today.getDate().toString().padStart(2, '0');
-            const date_string = `${year}-${month}-${day}`;
-            $("#poll-date").flatpickr({
-                defaultDate: date_string
-            });
-        {% endif %}
-        setEventHandlers();
-        $("#new-poll-form").on("change click", submitErrorChecks);
-        $("#edit-poll-form").on("change click", submitErrorChecks);
-        $("#new-poll-form").submit(function() {
-            $("#poll-form-submit").prop("disabled", true);
-            return true;
-        });
-        let submit_form = false;
-        $("#edit-poll-form").submit(function(event) {
-            if (!submit_form) {
-                if ((poll_type != "single-response"
-                    && ($('#poll-type-single-response-single-correct').is(':checked')
-                        || $('#poll-type-single-response-multiple-correct').is(':checked')
-                        || $('#poll-type-single-response-survey').is(':checked')))
-                || (poll_type != "multiple-response"
-                    && ($('#poll-type-multiple-response-exact').is(':checked')
-                        || $('#poll-type-multiple-response-flexible').is(':checked')
-                        || $('#poll-type-multiple-response-survey').is(':checked')))) {
-                    // don't submit yet
-                    event.preventDefault();
-                    // send warning
-                    if (confirm("Changing the poll question type may lead to data of students' answers being " +
-                               "lost if students have already begun submitting responses to this poll. " +
-                               "Are you sure you want to change the type of this poll?")) {
```

Comment:
We probably want this warning message back in case instructors accidentally change the poll type after responses have been submitted to it, since it's undefined behavior.

Reply:
The warning was actually there, but the logic wasn't being triggered correctly.  Changes made.